scripts/jenkins/Jenkinsfile

```diff
@@ -773,24 +817,50 @@ def xunitUpload() {
     ])
 }
 
+/* Create a new Docker Image Description
+ *
+ * @param name Name of the image, will be extended with registry, a common
+ *             prefix and a tag
+ * @param tagFun Closure describing how the tag should be formatted
+ * @param context Context for the docker build
+ * @param file Path to Dockerfile
+ * @param autobuild If it should be automatically build at the start of the
+ *                  Jenkins run
```

Comment:
Alternatively it is not build?

Reply:
automatically. can always build it in a stage manually

Reply:
used for the homepage builds for example